changes/367.housekeeping

```diff
@@ -0,0 +1 @@
+Update project to support python 3.12
```

Comment:
I think this should be a `changed` or `added` category so it's more visible.

```suggestion
Added support for python 3.12.
```

Reply:
Good idea. Done.